### PR TITLE
Objects Diagnostic

### PR DESCRIPTION
**Pull request by cob0910**
added answers to diagnostic

Comfort: 4
Clarity: 4

I used Google and my class notes to help with this diagnostic.

### CODE REVIEW

File: lib/diagnostic.js [hunk 1]
```diff
@@ -38,19 +47,38 @@ let owner = null;
...
-  /* Your answer here */
-};
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
+
+  this.vocalize = vocalize;
+  };
```
Comment (bernardlee) [inline lib/diagnostic.js:73]: This brace closes ```function (name, ..., vocalization) {```, but the next one has no matching opening bracket and breaks the script.

File: lib/diagnostic.js [hunk 2]
```diff
@@ -38,19 +47,38 @@ let owner = null;
...
+}
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function () {
+  return this.vocalize;
```
Comment (bernardlee) [inline lib/diagnostic.js:79]: This should be ```return this.vocalization;```.  In its current state, it will just return the function itself.

**Comment by bernardlee** [comment]
4/5

File: lib/diagnostic.js [hunk 3]
```diff
@@ -38,19 +47,38 @@ let owner = null;
...
-let pet = null;
+let pet = {
+  name: 'Buddy',
+  species: 'Cat',
+  breed: 'shorthair',
+  vocalization: 'Meow',
+
+  vocalize: function() {
+    return 'Buddy says Meow';
+  },
```
Comment (BenGitsCode) [inline lib/diagnostic.js:58]: you don't need to hardcode the vocalization. 

Returning 
```js 
return this.vocalization;  
```
will return "Meow"

File: lib/diagnostic.js [hunk 4]
```diff
@@ -38,19 +47,38 @@ let owner = null;
...
-const Pet = function () {
-  /* Your answer here */
-};
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
+
+  this.vocalize = vocalize;
```
Comment (BenGitsCode) [inline lib/diagnostic.js:72]: You don't need this line.

File: lib/diagnostic.js [hunk 5]
```diff
@@ -38,19 +47,38 @@ let owner = null;
...
+  };
+}
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function () {
+  return this.vocalize;
+};
+
```
Comment (BenGitsCode) [inline lib/diagnostic.js:81]: Close!
```js 
const vocalize = function () {
  return 'bark';
};

Pet.prototype.vocalize = vocalize;
```

**Pull request closed without merging**